app/js/main.js

```diff
@@ -339,7 +339,15 @@ function setupMeteringDemo(subscriptions) {
           },
         });
       })
-      .catch(() => false)
+      .catch(() => {
+        // If something goes wrong during the Regwall flow...
+
+        // Re-enable scrolling.
```

Comment:
I think it's okay, because the Offers carousel pops up right afterwards. Scenic doesn't block scrolling for the Offers carousel  currently (https://scenic-2017.appspot.com/1?showOffers)